google/cloud/security/common/data_access/sql_queries/snapshot_cycles_sql.py

```diff
@@ -24,6 +24,7 @@
         `complete_time` datetime DEFAULT NULL,
         `status` enum('SUCCESS','RUNNING','FAILURE',
                       'PARTIAL_SUCCESS','TIMEOUT') DEFAULT NULL,
+        `has_all_data` bool DEFAULT NULL,
```

Comment:
I think that effect of renaming the ```status``` column will be minimal.  It seems only IAM explain will be impacted, so I won't handle the schema change just for it.  @felixbb, is this okay?

If we want to change the naming to ```pipeline_data_status```, it might be better to have ```SUCCESS``` and ```FAILURE``` in order to be consistent with ```pipeline_execution_status```.  Otherwise, it seems odd if one is an enum, and the other is a bool, even though they are named the same?  @mirons-google